PyInstaller/building/api.py

```diff
@@ -253,25 +267,33 @@ def assemble(self):
                         # happen if they come from different sources (eg. once from
                         # binary dependence, and once from direct import).
                         if inm in seenInms:
-                            logger.warning('Two binaries added with the same internal name.')
+                            logger.warning(
+                                'Two binaries added with the same internal name.'
+                            )
```

Comment:
Any idea why this was split? The one-liner fits the (new) 120-character line limit, doesn't it?

Reply:
Huh it appears I accidentally deleted the yapf config. Ran without the line length lift.

Reply:
Aha, I'll wait for the updated version with increased line length, then.